.travis.yml

```diff
@@ -23,11 +23,12 @@ env:
 - TOX_ENV=py26e2e
 - TOX_ENV=py27ssh
 - TOX_ENV=py26ssh
+- TOX_ENV=docs
 install:
   - pip install --upgrade pip
   - pip install --upgrade setuptools
   - pip install tox
 script:
   - pip --version
   - tox --version
-  - tox -e $TOX_ENV
+  - PYTEST_PROCESSES=1 tox -e $TOX_ENV
```

Comment:
Yes AppVeyor and yes concurrency. :) AppVeyor works fine with the `-n auto` defaults, likely because it's running in a restricted VM. The problem is only with Travis, because the OS reports a very large amount of CPUs (12 in our case) but the container is extremely restricted, and thus we get failures.